### ISSUE TITLE
CustomElement Zoom Factor

### ISSUE THREAD

**Issue by lottilux** [umlet/umlet#596]
There is an issue with the Custom Elements and zoom factor.
Once the zoom deviates from 100% the connectors get lost (see screenshot).
![image](https://user-images.githubusercontent.com/19551529/96713688-45676b00-13a1-11eb-964d-f2e29eabf018.png)

**Comment by 911992** [comment]
Could you please share your custom element definition?

**Comment by afdia** [comment]
You're right, this gist contains a sample uxf file: https://gist.github.com/afdia/d159651f554e652e6acdca1fb1a044ad

- When opened it works
-  at 100% zoom
- When zooming to 90% it stops sticking
- When at 90% zoom and the right element is moved 4 spaces to the right on the grid, it starts sticking again although they should not be connected

All of this looks like the "sticking-calculation zoom factor" is not aligned with the custom "element zoom factor".

**Comment by lottilux** [comment]
Here is my CustomElement Example uxf file.
[Bug_569.zip](https://github.com/umlet/umlet/files/5428596/Bug_569.zip)
